app/mailers/course_update_email_mailer.rb

```diff
@@ -0,0 +1,29 @@
+class CourseUpdateEmailMailer < GovukNotifyRails::Mailer
+  def send_course_update_email(course, attribute_changed)
+    set_template(Settings.govuk_notify.course_update_email_template_id)
+
+    set_personalisation(
+      provider_name: course.provider.provider_name,
+      course_name: course.name,
+      course_code: course.course_code,
+      attribute_changed: attribute_changed,
+      attribute_change_datetime: parse_update_datetime(course.updated_at),
+      course_url: create_course_url(course)
+    )
+
+    mail(to: course.accrediting_provider.email)
+  end
+
+  private
+
+  def create_course_url(course)
+    "https://www.publish-teacher-training-courses.service.gov.uk" \
```

Comment:
The base URL should come from settings as it will differ in different environments.